prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/ConstantResolver.kt

```diff
@@ -73,6 +75,33 @@ object ConstantSubstitutor {
     }
 }
 
+/**
+ * Custom FHIR commands created by report stream to help map from FHIR -> HL7
+ * only used in cases when the same logic couldn't be accomplished using the FHIRPath
+ */
+object CustomFHIRCommands {
+
+    /**
+     * Converts a FHIR AddressUse enum to the HL7 v2.5.1 - 0190 - Address type
```

Comment:
You need proper kdoc for all these functions, properties, etc.  Maybe review your changes before creating the PR to make sure you cover any code changes with proper kdoc.